src/test/java/org/junit/internal/runners/statements/FailOnTimeoutTest.java

```diff
@@ -212,7 +212,7 @@ private void notTheRealCauseOfTheTimeout() {
     public void threadGroupNotLeaked() throws Throwable {
```

Comment:
This test is being replaced in pull 1687

Based on the CI run, I suspect that it's flaky anyway.